.github/workflows/ci.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.19
+          go-version: '1.20'
```

Comment:
It is recommended to merge the commit, there are too many  commit id.

Reply:
All the commits will be squashed into one when this PR is merged into the master.